ci-operator/step-registry/storage/conf/csi-optional/aws-efs/add-annotations/storage-conf-csi-optional-aws-efs-add-annotations-commands.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+set -x
+
+# For disconnected or otherwise unreachable environments, we want to
+# have steps use an HTTP(S) proxy to reach the API server. This proxy
+# configuration file should export HTTP_PROXY, HTTPS_PROXY, and NO_PROXY
+# environment variables, as well as their lowercase equivalents (note
+# that libcurl doesn't recognize the uppercase variables).
+if test -f "${SHARED_DIR}/proxy-conf.sh"
+then
+	# shellcheck disable=SC1091
+	source "${SHARED_DIR}/proxy-conf.sh"
+fi
+
+# logger function prints standard logs
+logger() {
```

Comment:
It's good to use a function and have a better format for logging. However, some echo statements in this script could be replaced with the logger for consistency in other parts of the script.